test/ml_methods/test_unsupervisedSklearnMethod.py

```diff
@@ -0,0 +1,75 @@
+import os
+import pickle
+import shutil
+from unittest import TestCase
+
+import numpy as np
+from scipy import sparse
+from sklearn.cluster import KMeans as SklearnKMeans
+
+from immuneML.IO.ml_method.MLMethodConfiguration import MLMethodConfiguration
+from immuneML.data_model.encoded_data.EncodedData import EncodedData
+from immuneML.environment.EnvironmentSettings import EnvironmentSettings
+from immuneML.ml_methods.Clustering.KMeans import KMeans
+from immuneML.util.PathBuilder import PathBuilder
+
+
+class TestUnsupervisedSklearnMethod(TestCase):
+    def test_load(self):
```

Comment:
this is an abstract class - no need to test the same thing? see the comment above?